dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/task/switchtask/SwitchParameters.java

```diff
@@ -74,21 +76,18 @@ public void setDependTaskList(List<SwitchResultVo> dependTaskList) {
         this.dependTaskList = dependTaskList;
     }
 
-    public List<String> getNextNode() {
+    public Long getNextNode() {
         return nextNode;
     }
 
     public void setNextNode(Object nextNode) {
-        if (nextNode instanceof String) {
-            List<String> nextNodeList = new ArrayList<>();
-            nextNodeList.add(String.valueOf(nextNode));
-            this.nextNode = nextNodeList;
-        } else if (nextNode instanceof Number) {
-            List<String> nextNodeList = new ArrayList<>();
-            nextNodeList.add(nextNode.toString());
-            this.nextNode = nextNodeList;
+        if (nextNode instanceof Long) {
+            this.nextNode = (Long) nextNode;
         } else {
-            this.nextNode = (ArrayList) nextNode;
+            List<String> nextNodes = (ArrayList) nextNode;
```

Comment:
<img width="749" alt="image" src="https://user-images.githubusercontent.com/31072303/218965398-a007d573-3e62-4d1d-99f6-5c251d60fea0.png">

An CaseError will occur if the nextNode is empty when `branch flow` is not set.

Reply:
I think it is possible. Can you submit a new issue to describe it